mtp_api/apps/mtp_auth/tests/utils.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+from django.utils import timezone
+from django.utils.crypto import get_random_string
+
+from oauth2_provider.models import Application, AccessToken
+from mtp_auth.constants import CASHBOOK_OAUTH_CLIENT_ID, \
+    BANK_ADMIN_OAUTH_CLIENT_ID, PRISONER_LOCATION_OAUTH_CLIENT_ID
+
+
+class AuthTestCaseMixin(object):
+    APPLICATION_ID_MAP = {
+        'PrisonerLocationAdmin': PRISONER_LOCATION_OAUTH_CLIENT_ID,
+        'BankAdmin': BANK_ADMIN_OAUTH_CLIENT_ID,
+        'PrisonClerk': CASHBOOK_OAUTH_CLIENT_ID
+    }
+
+    def _get_http_authorization_token_for_user(self, user):
+        group = user.groups.first()
```

Comment:
Would pretty much get covered by https://github.com/ministryofjustice/money-to-prisoners-api/pull/23#discussion_r37415654 I guess?